_minutes/2024-02-15-wecg.md

```diff
@@ -0,0 +1,158 @@
+ * [rob] web_accessible_resources patterns are only limited in Chrome. Would prefer to address this issue by having the extension inject on all pages of the origin and observe History/Navigation API calls itself. Major problem is that we can't un-run a content script.
+ * [oliver] I left a comment with similar things. Offering this at the platform level has a number of issues.
+ * [simeon] The resolution here is to document this aspect on MDN.
+
+[Issue 527](https://github.com/w3c/webextensions/issues/527): chrome.scripting.executeScript doesn't resolve/reject for frozen state tabs
+
+ * [simeon] When scripting.executeScript is run on a frozen tab, execution does not happen immediately.
+ * [rob] What is “frozen”?
+ * [oliver] When a tab is put in a tab group, the tab is frozen, I don't know the specific meaning with it.
+ * [simeon] bfcache also has a similar issue.
+ * [oliver] The request here is, if the execution cannot succeed immediately, should the call wait or reject immediately?
+ * [simeon] Perhaps we should add a flag to the tab to signal whether the tab is able to handle script injection.
+ * [rob] Prone to race conditions. Prefer to expose another property on the options bag. This concern also applies to other cases like permission requests (e.g. in Safari where extension API calls are suspended until host permissions have been granted). Looking at the broader area of request with a delayed response, we should consider how to signal that a call is suspended and/or aborted. Complete API proposal would be welcome.
+
+[Issue 540](https://github.com/w3c/webextensions/pull/540): Add userScripts.execute() API proposal
+
+ * [emilia] Code injection using user scripts API, programmatically.
+ * [rob] I will take a closer look after the meeting; note that this has already been discussed before at [issue 477](https://github.com/w3c/webextensions/issues/477).
+
+[Issue 433](https://github.com/w3c/webextensions/issues/433): Inconsistency: Alarms API time slipping
+
+ * [anton] Chrome 123 uses wall clock time.
+ * [anton] Should we keep the issue open for fixing Firefox sleep behavior and keeping a consistent schedule for repeating alarms?
+ * [anton] We can close the issue or keep it around for Firefox.
+ * [oliver] Another issue is firing the alarm on a consistent schedule, when repeating.
+ * [anton] That is a separate issue.
+ * [oliver] From what we discussed before, the alarm should keep going when the device resumes.
+ * [anton] Suggest a separate issue to track that.
+ * [simeon] Can wpt cover this scenario in tests?
+ * [patrick] I don't know from the top of my head. I guess that it might not, depending on the hooks around suspension.
+
+[Issue 401](https://github.com/w3c/webextensions/issues/401): Expose SHA256 value of DownloadItem
+
+ * [anton] Q: should the has be an ArrayBuffer or a hex string? Hex string is most common use, ArrayBuffer is more consistent with the crypto.digest API.
```

Comment:
Should the what? be an ArrayBuffer? Something went off the rials here.

Reply:
+1 - assume it is "Should this be an"

Reply:
```suggestion
 * [anton] Q: should the hash be an ArrayBuffer or a hex string? Hex string is most common use, ArrayBuffer is more consistent with the crypto.digest API.
```

One letter was missing.